src/main/java/seedu/address/model/client/ClientId.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.client;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.UUID;
```

Comment:
`UUID` may not be appropriate in our project - random UUID's will be generated every run of `DevEnable`. May leave this out as I will be implementing it.

Reply:
Ok! Will remove and push a commit! So the `ClientId` is set as part of the constructor of `Client` like `IssueId`, `ProjectId` etc. Thanks for the clarification!